docs/docs-content/integrations/ngrok.md

```diff
@@ -0,0 +1,142 @@
+---
+sidebar_label: "ngrok"
+title: "ngrok"
+description: "Learn about using ngrok Kubernetes Ingress to access applications and services in Spectro Cloud."
+hide_table_of_contents: true
+type: "integration"
+category: ["ingress", "kubernetes"]
+sidebar_class_name: "hide-from-sidebar"
+logoUrl: ""
+tags: ["packs", "ngrok", "network", "kubernetes"]
+---
+
+The ngrok [Ingress Controller for Kubernetes](https://github.com/ngrok/kubernetes-ingress-controller) is the official
+controller for adding public and secure ingress traffic to your k8s services. This open source [Ingress
+Controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers) works with Spectro Cloud to
+provide ingress to your applications, APIs, or other services while also offloading network ingress and middleware
+execution to ngrok's platform.
+
+# Versions Supported
+
+- 0.9.0
+
+## Prerequisites
+
+- An active [ngrok account](https://ngrok.com/signup).
+  - Your ngrok authentication token (`AUTHTOKEN`), which you can find under under [**Your
+   Authtoken**](https://dashboard.ngrok.com/get-started/your-authtoken) in the ngrok dashboard.
+  - An API key (`API_KEY`), which you can generate in the [**API** section](https://dashboard.ngrok.com/api) of the
+    ngrok dashboard.
+- A static subdomain (`NGROK_DOMAIN`), which you can obtain by navigating to the [**Domains**
+   section](https://dashboard.ngrok.com/cloud-edge/domains) of the ngrok dashboard and clicking **Create Domain** or
+   **New Domain**.
+
+## Parameters
```

Comment:
@joelhans for this section, please create a markdown table containing all the required parameters; API_KEY, AUTHTOKEN, and DOMAIN.
This table should be placed at the top of the parameters section. The table requires the two following columns; Parameter, Description

This section is often associated with the YAML of the pack itself. Are there other common parameters worth highlighting an placing in the markdown table? you don't have add every single parameter available in the YAML, just the ones worth being aware off. You can link to these resources that you all have created:
 - [Common Helm Overrides](https://github.com/ngrok/kubernetes-ingress-controller/blob/main/docs/deployment-guide/common-helm-k8s-overrides.md)
 -  [ngrok Kubernetes Ingress Controller Documentation](https://github.com/ngrok/kubernetes-ingress-controller/blob/main/docs/README.md#ngrok-kubernetes-ingress-controller-documentation)

![CleanShot 2023-09-27 at 07 31 00](https://github.com/spectrocloud/librarium/assets/29551334/9d7d1bf4-ad6a-4942-94f4-bbc9c38a8051)

Reply:
Thanks for the tip here. I have added the *required* parameters into this table and then linked to resources where users could find more advanced configurations, like fan-outs, which mostly work via adding additional `path` parameters.